src/cpu.rs

```diff
@@ -178,4 +180,36 @@ impl Speed for CPU {
     fn render(&self) -> String {
         render_cpu(self)
     }
+
+    fn random() -> CPU {
+        let mut rng = rand::thread_rng();
+        CPU {
+            name: "TEST__0".to_string(),
+            number: rng.gen_range(0..100),
+            max_freq: rng.gen_range(0..100000),
+            min_freq: rng.gen_range(0..10000),
+            cur_freq: rng.gen_range(0..100000),
+            cur_temp: rng.gen_range(0..100000),
+            cur_usage: rng.gen::<f32>(),
+            gov: if rng.gen_bool(0.5) {
+                "powersave".to_string()
+            } else {
+                "performance".to_string()
+            },
+        }
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn cpu_random_unit_test() {
+        let cpu_1 = CPU::random();
+        let cpu_2 = CPU::random();
+
+        assert_ne!(cpu_1.cur_temp, cpu_2.cur_temp);
```

Comment:
I don't understand this test, what happens if both cpus roll the same number? What is the point of filling it with random information.

Reply:
@JakeRoggenbuck ^

Reply:
@JakeRoggenbuck